src/main/webapp/add-account.html

```diff
@@ -9,20 +9,20 @@
   </head>
   <body onload="javascript:populateCustomerList()">
     <div class="drop_menu">
-			<button class="menu_button">Page Menu</button>
```

Comment:
Thanks for fixing this formatting problem. Cloudshell can be a bit weird with having formatting look good on its site but bad on others.